meshtastic/tests/test_mesh_interface.py

```
@@ -1754,11 +1791,111 @@ def test_on_response_position_success_and_routing_error(
+        stdout_handler.setLevel(logging.INFO)
+        handler_logger.addHandler(stdout_handler)
+        assert (
+            mesh_interface_module._logger_has_visible_info_handler(handler_logger)
+            is True
+        )
+
+        handler_logger.removeHandler(stdout_handler)
+        stdout_handler.close()
+
+        class _RichLikeHandler(logging.Handler):
+            def __init__(self, stream: Any) -> None:
```

Ruff (0.15.4)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `stream`

(ANN401)